src/utilities/extendSchema.js

```diff
@@ -280,6 +379,18 @@ export function extendSchema(
     });
   }
 
+  function extendArgsMap(
+    args: GraphQLArgument[],
```

Comment:
It accepts and returns an array so it probably shouldn't have `Map` in its name.